### PR TITLE
Mmi 5637 ethena missing farming product

### PR DESCRIPTION
**Pull request by bergarces**

### CODE REVIEW

File: packages/adapters-library/src/adapters/aave-v2/products/a-token/aaveV2ATokenAdapter.ts [hunk 1]
```diff
@@ -8,7 +8,6 @@ export class AaveV2ATokenPoolAdapter extends AaveBasePoolAdapter {
   adapterSettings = {
     enablePositionDetectionByProtocolTokenTransfer: true,
     includeInUnwrap: true,
-    version: 3,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/aave-v2/products/a-token/aaveV2ATokenAdapter.ts:11]: Missed in previous PR.

File: packages/adapters-library/src/adapters/etherfi/products/e-eth/etherFiEEthAdapter.ts [hunk 2]
```diff
@@ -33,7 +33,6 @@ export class EtherFiEEthAdapter implements IProtocolAdapter {
   helpers: Helpers
 
   adapterSettings = {
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/etherfi/products/e-eth/etherFiEEthAdapter.ts:36]: Missed on previous PR.

File: packages/adapters-library/src/adapters/etherfi/products/l2/etherFiL2Adapter.ts [hunk 3]
```diff
@@ -55,7 +55,6 @@ export class EtherFiL2Adapter implements IProtocolAdapter {
   helpers: Helpers
 
   adapterSettings = {
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/etherfi/products/l2/etherFiL2Adapter.ts:58]: Missed on previous PR.

File: packages/adapters-library/src/adapters/etherfi/products/liquid/etherFiLiquidAdapter.ts [hunk 4]
```diff
@@ -43,7 +43,6 @@ export class EtherFiLiquidAdapter implements IProtocolAdapter {
   helpers: Helpers
 
   adapterSettings = {
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/etherfi/products/liquid/etherFiLiquidAdapter.ts:46]: Missed on previous PR.

File: packages/adapters-library/src/adapters/etherfi/products/we-eth/etherFiWeEthAdapter.ts [hunk 5]
```diff
@@ -34,7 +34,6 @@ export class EtherFiWeEthAdapter implements IProtocolAdapter {
   helpers: Helpers
 
   adapterSettings = {
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/etherfi/products/we-eth/etherFiWeEthAdapter.ts:37]: Missed on previous PR.

File: packages/adapters-library/src/adapters/mountain-protocol/products/usdm/mountainProtocolUsdmAdapter.ts [hunk 6]
```diff
@@ -19,7 +19,6 @@ export class MountainProtocolUsdmAdapter extends SimplePoolAdapter {
   adapterSettings = {
     enablePositionDetectionByProtocolTokenTransfer: true,
     includeInUnwrap: true,
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/mountain-protocol/products/usdm/mountainProtocolUsdmAdapter.ts:22]: Missed on previous PR.

File: packages/adapters-library/src/adapters/mountain-protocol/products/wusdm/mountainProtocolWUsdmAdapter.ts [hunk 7]
```diff
@@ -32,7 +32,6 @@ export class MountainProtocolWUsdmAdapter extends SimplePoolAdapter {
   adapterSettings = {
     enablePositionDetectionByProtocolTokenTransfer: true,
     includeInUnwrap: true,
-    version: 2,
```
Comment (bergarces) [inline packages/adapters-library/src/adapters/mountain-protocol/products/wusdm/mountainProtocolWUsdmAdapter.ts:35]: Missed on previous PR.

File: packages/adapters-library/src/adapters/zerolend/contracts/abis/protocol-data-provider.json
Comment (bergarces) [inline packages/adapters-library/src/adapters/zerolend/contracts/abis/protocol-data-provider.json:1]: They had copied the contracts folder from aave and deleted the abi for this contract, so I was getting an error when building-types and had to add it back.

**Review by jpsains — APPROVED** [review]

**Pull request merged**